config/auth_default.go

```diff
@@ -13,6 +13,7 @@ var (
 	authProviders = []CredentialsProvider{
 		PatCredentials{},
 		BasicCredentials{},
+		BricksCliCredentials{},
```

Comment:
As is, this should be `BricksOAuthCredentials`. Currently it implies it can do anything.